include/RAJA/policy/loop/atomic.hpp

```diff
@@ -27,7 +27,7 @@
 namespace RAJA
 {
 
-using loop_atomic = seq_atomic;
+// Delete?
```

Comment:
I think we'll need a backward compatibility solution for this. Maybe something like`using loop_atomic = policy::loop::loop_atomic;`.

Reply:
Makes sense to me, I'll go ahead and change that.